src/registry/registry.ts

```diff
@@ -249,7 +249,7 @@ export function getRegisteredExpressionHelpers() {
 }
 
 function getFormsStore(): FormsRegistryStoreState {
-  return getGlobalStore<FormsRegistryStoreState>(OHRIFormsStore, {
+  return getGlobalStore<FormsRegistryStoreState>(FormsStore, {
```

Comment:
This is a weird way of using a global store. Normally, we'd create a store in a variable like:

```ts
const formsStore = createGlobalStore<FormsRegistryStoreState>(FormsStore, /* ... */);
```

Then, you could create synchronous handlers, e.g.,

```ts
export function getRegisteredExpressionHelpers() {
  return formsStore.getState().expressionHelpers;
}
```

But, even better, since this is React, you can just turn everything into hooks:

```ts
export const useRegisteredExpressionHelpers = useStore(formsStore, (state) => state.expressionHelpers);
```

Setters can then be implemented more sanely like this:

```ts
export function registerFieldValidator(registration: ComponentRegistration<FormFieldValidator>) {
  formsStore.set((state) => state.fieldValidators.push(registration));
}
```